app/Http/Controllers/TechniqueController.php

```diff
@@ -12,10 +12,18 @@ class TechniqueController extends Controller
 {
     public function index(): View
     {
+        $techniques = Technique::all();
         $viewData = [];
         $viewData['title'] = trans('app.titles.home');
         $viewData['selected'] = 'techniques';
-        $viewData['techniques'] = Technique::all();
+        $viewData['techniques'] = $techniques;
+
+        $diccionarioDeReviews = [];
```

Comment:
La idea es que el codigo esté en un mismo idioma
```suggestion
        $reviewsDictionary= [];
```
o 
```suggestion
        $reviews = [];
```